LoginForm.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useFormik } from 'formik';
+import { useHistory } from 'react-router-dom';
+
+function LoginForm() {
+  let history = useHistory();
+
+  const formik = useFormik({
+    initialValues: {
+      email: '',
+      password: '',
```

Comment:
The onSubmit function should have logic to handle the login request to the API. The current implementation just redirects to the dashboard without any validation which is a security risk.